README.md

```diff
@@ -1,47 +1,63 @@
-# cbioportal-docker
+# cbioportal-docker @ The Hyve
```

Comment:
:)

Reply:
I wrote that heading and first paragraph to demonstrate how the approach differs from that documented in the [cBioPortal/cbioportal](https://github.com/cBioPortal/cbioportal/blob/v1.10.1/Dockerfile) repo itself. If we're making this approach a community effort but maintaining two separate workarounds for the limitations in the cBioPortal build process, we should probably rephrase this.